app/routes/farmer-apply/declaration.js

```diff
@@ -87,18 +88,25 @@ module.exports = [{
       }
     },
     handler: async (request, h) => {
-      session.setFarmerApplyData(request, declaration, true)
-
       const application = session.getFarmerApplyData(request)
-      await sendMessage(application, applicationRequestMsgType, applicationRequestQueue, { sessionId: request.yar.id })
-      const response = await receiveMessage(request.yar.id, applicationResponseQueue)
-      console.info('Response received:', util.inspect(response, false, null, true))
+      let applicationReference = application?.reference
```

Comment:
Should this be `application[reference]` so it uses the reference variable rather than the key value `reference`